peldam_pie/lib/style.dart

```diff
@@ -0,0 +1,12 @@
+import 'dart:ui';
+
```

Comment:
These are in the wrong folder. You have this in peldam_pie/lib/style.dart instead of lib/style.dart. 
Also can you please remove the peldam_pie folder. It's unnecessary.

Reply:
that is the external library for drawing pie so customize it. it a seperate folder.. do you get

Reply:
have added the folder to thirdparty folder..

Reply:
have added the folder to thirdparty folder..